zeek-client

```diff
@@ -103,6 +103,10 @@ class ClientConfig(configparser.ConfigParser):
 
                 # Delay between our on connect attempts.
                 'connect_retry_delay_secs': 0.25,
+
+                # Whether we pretty-print JSON output by default. Use
+                # --pretty-json command-line flag to override.
+                'pretty_json': False,
```

Comment:
Maybe make this the default for JSON output?

Reply:
Good one, will do.